xbmc/pvr/channels/PVRChannelGroupInternal.cpp

```diff
@@ -36,10 +36,11 @@ using namespace PVR;
 using namespace EPG;
 
 CPVRChannelGroupInternal::CPVRChannelGroupInternal(bool bRadio) :
-  CPVRChannelGroup(bRadio, bRadio ? PVR_INTERNAL_GROUP_ID_RADIO : PVR_INTERNAL_GROUP_ID_TV, g_localizeStrings.Get(19287)),
   m_iHiddenChannels(0)
 {
   m_iGroupType = PVR_GROUP_TYPE_INTERNAL;
+  m_bRadio = bRadio;
+  m_strGroupName = g_localizeStrings.Get(19287);
```

Comment:
has been initialzers before, but I had to revert this because of compile issues (the 3 vars are members of the inherited class and whyever the compiler doesn't wan't then as initializers...) I will change for sure if you tell me the trick (never done this before, maybe CPRVChannelGroup::m_bRadio(bRadio)?